lib/src/rules/use_build_context_synchronously.dart

```diff
@@ -85,6 +85,12 @@ class _AwaitVisitor extends RecursiveAstVisitor {
   void visitAwaitExpression(AwaitExpression node) {
     hasAwait = true;
   }
+
+  @override
+  visitBlockFunctionBody(BlockFunctionBody node) {
```

Comment:
You probably also want to visit `ExpressionFunctionBody` to stop at the `=>` form.

Reply:
You are right, thanks!